linux/snowagent.config

```diff
@@ -27,72 +27,112 @@
     <!-- Configuration for File System scan-->
     <Software>
         <Include>
-            <Path recursive="true" unconditionally="false">/*</Path>
+			<!-- Only include necessary folders for the file scan-->
+            <Path recursive="true" unconditionally="false">/etc/*</Path>
+			<Path recursive="true" unconditionally="false">/var/*</Path>
```

Comment:
can we keep the same indentation?

Reply:
Probably tabs vs spaces?

Reply:
Seems like GitHub assigned different tab size for my changes.
I edited in the Git editor this time and it seemed ok

Reply:
still not resolved. please activate displaying whitespaces and tabs in n++ like "menu => View => Show symbol => Show white space and tab" and then just press space instead of tab.

Reply:
We usually use tabs to indent....not sure what this file uses though.

Reply:
I used tab, but the file was using spaces.
Should be fixed now